testutil/internal.go

```diff
@@ -0,0 +1,33 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package testutil
+
+import (
+	"github.com/snapcore/snapd/dirs"
+)
+
+// MockInternalCmdPath mocks the path used for lookups of internal commands,
+// such as snap-confine, that are not on PATH.
+func MockInternalCmdPath(path string) (restore func()) {
```

Comment:
See above, not sure we need this, given that we generally do something like "dirs.SetRootDir(c.MkDir())" in the tests, so this is usually already pointing to a place that we can mock around plus it gets restored for free usually.